composer.lock

```diff
@@ -1225,16 +1225,16 @@
         },
         {
             "name": "drupal/core",
-            "version": "9.4.4",
+            "version": "9.4.5",
```

Comment:
Apparently, when review is required I need to actually leave a comment on the code ...

Reply:
... and resolve it.